packages/nimble-components/src/toggle-button/styles.ts

```diff
@@ -5,87 +5,106 @@ import {
     borderHoverColor,
     borderWidth,
     fillSelectedColor,
+    fillSelectedRgbPartialColor,
     smallDelay
 } from '../theme-provider/design-tokens';
 import { styles as buttonStyles } from '../patterns/button/styles';
 
 export const styles = css`
     ${buttonStyles}
 
-    .control[aria-pressed='true'] {
-        background-color: ${fillSelectedColor};
-        border-color: ${fillSelectedColor};
-        position: relative;
-        transition: box-shadow ${smallDelay} ease-in-out,
-            border-color ${smallDelay} ease-in-out,
-            background-size ${smallDelay} ease-in-out;
-        background-size: 100% 100%;
-        background-repeat: no-repeat;
-        background-position: center;
-    }
+    @layer default {
+        .control[aria-pressed='true'] {
+            background-color: transparent;
+            background-image: linear-gradient(
+                ${fillSelectedColor},
+                ${fillSelectedColor}
+            );
+            border-color: rgba(${fillSelectedRgbPartialColor}, 0.3);
+            position: relative;
+            transition: box-shadow ${smallDelay} ease-in-out,
+                border-color ${smallDelay} ease-in-out,
+                background-size ${smallDelay} ease-in-out;
+            background-size: 100% 100%;
+            background-repeat: no-repeat;
+            background-position: center;
+        }
 
-    .control[aria-pressed='true']:hover {
-        border-color: ${borderHoverColor};
-        box-shadow: 0px 0px 0px ${borderWidth} ${borderHoverColor} inset;
-        background-image: linear-gradient(
-            ${fillSelectedColor},
-            ${fillSelectedColor}
-        );
-        background-size: calc(100% - 4px) calc(100% - 4px);
+        .control[aria-pressed='true']::before {
+            content: '';
+            position: absolute;
+            width: 100%;
+            height: 100%;
+            pointer-events: none;
+            box-sizing: border-box;
+            outline: 0px solid transparent;
+            background-clip: content-box;
+            transition: outline ${smallDelay} ease-in-out;
+        }
```

Comment:
These styles are completely defined in buttonStyles already it looks like. We can get rid of this altogether right?